monasca_installer/monasca_config.yaml.example

```diff
@@ -47,19 +47,28 @@ api_influxdb_password: PLEASE-GENERATE
 api_mysql_user: monapi
 api_mysql_password: PLEASE-GENERATE
 
-keystone_admin: admin
-keystone_admin_password: pass1234
-keystone_admin_token: ADMINTOKEN
-keystone_project_admin: monasca-agent
-keystone_project_admin_password: pass1234
-keystone_host: 10.22.156.19
-keystone_url: http://10.22.156.19:35357/v3
+all_config:
+  monasca_api_host: mon-ae1test-monasca01.useast.hpcloud.net
+  monasca_api_url: https://{{monasca_api_host}}:8080/v2.0
```

Comment:
monasca_api_url is repeated. Also if the monasca_api_host is only used here it can be collapsed together.